dev-packages/node-integration-tests/suites/tracing/requests/fetch-no-tracing/test.ts

```diff
@@ -3,31 +3,27 @@ import { createRunner } from '../../../../utils/runner';
 import { createTestServer } from '../../../../utils/server';
 
 conditionalTest({ min: 18 })('outgoing fetch', () => {
-  test('outgoing fetch requests are correctly instrumented without tracesSampleRate', done => {
-    expect.assertions(15);
+  test('outgoing fetch requests are correctly instrumented with tracing disabled', done => {
+    expect.assertions(11);
 
     createTestServer(done)
       .get('/api/v0', headers => {
         expect(headers['sentry-trace']).toEqual(expect.stringMatching(/^([a-f0-9]{32})-([a-f0-9]{16})$/));
         expect(headers['sentry-trace']).not.toEqual('00000000000000000000000000000000-0000000000000000');
         expect(headers['baggage']).toEqual(expect.any(String));
-        expect(headers['__requestUrl']).toBeUndefined();
```

Comment:
this is actually not needed anymore, we do not set this anywhere, removed it in all places.